src/components/CategoriesList.js

```diff
@@ -8,7 +8,7 @@ export const CategoriesList = (props) => {
   return (
     <div className="Categories-list"> 
       {
-        categories.map(category => <Category name={category.name} image={category.image}/>) 
+        categories.map(category => <Category key={category.name} name={category.name}/>) 
```

Comment:
@cgolinski "When you run this code, you'll be given a warning that a key should be provided for list items. A "key" is a special string attribute you need to include when creating lists of elements. We'll discuss why it's important in the next section." from https://facebook.github.io/react/docs/lists-and-keys.html